ppp_questionparsing_grammatical/dependencyTree.py

```diff
@@ -15,7 +14,7 @@ class Word:
     def __init__(self, word, index, pos=None):
         self.word = word    # string that represents the word
         self.index = index  # position in the sentence
-        self.pos = pos      # Part Of Speech tag (verb, noun, ...)
+        self.pos = pos or ""  # Part Of Speech tag (verb, noun, ...)
```

Comment:
Is it relevant to have `pos=None` and `self.pos = pos or ""`? I think we should keep None when there is no pos.